examples/basic/variables.tf

```diff
@@ -1,33 +1,35 @@
-########################################################################################################################
-# Input variables
-########################################################################################################################
+####################
+# Cluster variables
+####################
+variable "cluster_id" {
+  type        = string
+  description = "The name or ID of the cluster to be used"
+  default     = "my-cluster-name"
+}
 
 variable "ibmcloud_api_key" {
   type        = string
-  description = "The IBM Cloud API Key"
+  description = "IBM Cloud API key"
   sensitive   = true
 }
 
-variable "region" {
+variable "ibmcloud_region" {
   type        = string
-  description = "Region to provision all resources created by this example"
+  description = "IBM Cloud region where all resources will be deployed"
   default     = "us-south"
 }
 
-variable "prefix" {
-  type        = string
-  description = "Prefix to append to all resources created by this example"
-  default     = "basic"
-}
-
-variable "resource_group" {
-  type        = string
-  description = "The name of an existing resource group to provision resources in to. If not set a new resource group will be created using the prefix variable"
-  default     = null
+####################
+# Optional variables
+####################
+variable "enable_ibm_cert_manager" {
```

Comment:
ideally we dont want to make examples flexible solutions. So I don't suggest you expose this variable in the example, but hard code it in the main.tf of the basic example